gen3.biodatacatalyst.nhlbi.nih.gov/etlMapping.yaml

```diff
@@ -136,12 +137,14 @@ mappings:
       - name: md5sum
       - name: file_name
       - name: file_size
+      - name: data_category
       - name: data_format
       - name: data_type
       - name: state
       - name: callset
       - name: bucket_path
       - name: source_node
+      - name: data_category
```

Comment:
looks like this is in here twice? Is that intentional?

Reply:
The automation just brought the same config from preprod:
https://github.com/uc-cdis/cdis-manifest/blob/master/preprod.gen3.biodatacatalyst.nhlbi.nih.gov/etlMapping.yaml
--
We can amend this in the next release as it seems to be harmless.
cc: @svburke